checklists/bronze.md

```diff
@@ -5,13 +5,16 @@
 
 ### Documentation
 
-- [ ] Brief description of what program does is listed.
-- [ ] An example of its usage is shown.
-  - typical usage
-- [ ] Description of required input parameters.
-  - Reasonable description (i.e. "Nifti of brain mask in MNI" vs. "An image file")
-- [ ] Description of output(s).
-
+- [ ] README provides link to documentation
```

Comment:
To the question in the PR itself, whether or not we require the software to be open source, a README is a very "GitHub" concept, so I may suggest that we instead say something like the following:
```suggestion
- [ ] Landing page (e.g., GitHub README, website) provides a link to documentation
```